modules/ossm-installation-activities.adoc

```diff
@@ -12,5 +12,5 @@
 
 * *OpenShift Elasticsearch* - (Optional) Provides database storage for tracing and logging with the {JaegerShortName}. It is based on the open source link:https://www.elastic.co/[Elasticsearch] project.
 * *{JaegerName}* - Provides distributed tracing to monitor and troubleshoot transactions in complex distributed systems. It is based on the open source link:https://www.jaegertracing.io/[Jaeger] project.
-* *Kiali* - Provides observability for your service mesh. Allows you to view configurations, monitor traffic, and analyze traces in a single console. It is based on the open source link:https://www.kiali.io/[Kiali] project.
+* *Kiali Operator provided by Red Hat* - Provides observability for your service mesh. Allows you to view configurations, monitor traffic, and analyze traces in a single console. It is based on the open source link:https://www.kiali.io/[Kiali] project.
```

Comment:
I feel like there's something in the IBM style guide about using "allows", but I can't recall off the top of my head.

Reply:
Yeah, I think it falls under [anthropomorphism](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism), so might be worth trying to reword this. E.g., "You can view configurations, monitor traffic, and analyze traces in a single Kiali console." or something...